METRICS.md

```diff
@@ -73,8 +73,8 @@ Here is an example of grafana dashboard importing.
 |             FlushDataTime              |      worker       |                                  FlushData means flush a disk buffer to disk.                                   |
 |             OpenStreamTime             |      worker       |            OpenStream means read a shuffle file and send client about chunks size and stream index.             |
 |             FetchChunkTime             |      worker       |                      FetchChunk means read a chunk from a shuffle file and send to client.                      |
-|           MasterPushDataTime           |      worker       |                       MasterPushData means handle pushdata of master partition location.                        |
-|           SlavePushDataTime            |      worker       |                        SlavePushData means handle pushdata of slave partition location.                         |
+|           PrimaryPushDataTime          |      worker       |                       PrimaryPushData means handle pushdata of primary partition location.                        |
+|           ReplicaPushDataTime          |      worker       |                        ReplicaPushData means handle pushdata of replica partition location.                         |
```

Comment:
Better mention this change in migration doc

Reply:
shall we put the change note in the section `Upgrading from 0.2 to 0.3`?

Reply:
> shall we put the change note in the section `Upgrading from 0.2 to 0.3`?

Yea

Reply:
this has been addressed in #1650